tutorials/metrics/metrics_file_regex_with_linenum.yml

```diff
@@ -0,0 +1,27 @@
+buildspecs:
+  metric_file_regex:
+    executor: generic.local.bash
+    type: script
+    description: capture result metric from file path
+    run: echo -e "HPCG result is INVALID with a GFLOP/s rating of=28.1215\n
+                  HPCG result is VALID with a GFLOP/s rating of=68.9888" > hpcg.txt
+    tags: tutorials
+    metrics:
+      second_line:
+        type: float
+        file_regex:
+          exp: '(\d+\.\d+)$'
+          linenum: 1
+          file: hpcg.txt
+      without_linenum:
+        type: float
+        file_regex:
+          exp: '(\d+\.\d+)$'
+          file: hpcg.txt
+    status:
+      assert_eq:
+        comparisons:
+        - name: second_line
```

Comment:
change this to `last_line` if you use `-1`

Reply:
👍